src/vaults/mainnet.json

```diff
@@ -10,6 +10,11 @@
       "slug": "beraborrow-bwbera",
       "stakeTokenAddress": "0x6751C09113a83a83a43829fD0b3BC0D7bdbe07bf"
     },
+    {
+      "beraRewardsVault": "0x597877Ccf65be938BD214C4c46907669e3E62128",
```

Comment:
this address points to the staked nectar token
https://berascan.com/address/0x597877Ccf65be938BD214C4c46907669e3E62128

please provide the correct `beraRewardsVault`